wotw_seedgen/areas.wotw

```diff
@@ -5553,11 +5566,11 @@ anchor InnerWellspring.DrainAreaEntrance at -1128, -3968:  # On the wall after d
       Water, Bash, Grenade=1, Grapple OR Glide
       Launch
     gorlek:
-      DoubleJump, Water OR TripleJump OR Dash OR Bash OR Glide OR Sword OR Hammer OR Damage=20
+      DoubleJump, TripleJump OR Dash OR Glide OR Sword OR Hammer OR Damage=20
       Bash, Dash, Sword OR Hammer
-      Bash, Grenade=1, Water OR Dash OR Damage=20
+      Bash, Grenade=1, Water OR Damage=20
       Bash, SentryJump=1, Water OR Damage=20
-      Water, WaterDash, Dash, Sword OR Hammer  # @validator Added path. Gorlek might want some variation of this.
+      Water, WaterDash, Dash, Sword OR Hammer
```

Comment:
Without DoubleJump or Bash, I find it difficult to work with the Skeeto and generally the enemies nearby.
Added a discord post as well. If we decide not to include this in gorlek, we should add a comment explaining that we only have paths with DoubleJump or Bash because it helps with the enemies.